### DISCUSSION TITLE
Which devices might be affected if we remove <queries>...</queries>?

### DISCUSSION THREAD

**Discussion by UmberFunk** [yasirkula/UnityNativeShare#142] [Q&A] [answered]
> In the worst case, if you are OK with NativeShare not working on some of the affected devices, then...

Which devices might be affected if we remove this? Or can you tell me a direction to Google that? I tried to Google it, but with no success. I have not done development on Android and was not able to guess good keywords for Google.

Upgrading Gradle didn't work for me. I think I will try again after the deadline when I will get back to the project. Was curious if the affected devices are not too much to leave it for now.

**Comment by yasirkula** [comment, marked as answer]
It's either all Android 11+ devices when API Level 30 or higher is targeted, or all Android devices when API Level 30 or higher is targeted. I also can't tell with certainty which one is the case.

https://developer.android.com/training/package-visibility

**Reply by UmberFunk** [reply]
The answer was quite informative! Thanks a lot for your help!
